app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java

```diff
@@ -93,6 +94,8 @@ public Fragment getItem(int position) {
                     return  ExperimentDocFragment.newInstance("L_halfWave.html");
                 if (experimentTitle.equals(context.getResources().getString(R.string.full_wave_rectifier)))
                     return ExperimentDocFragment.newInstance("M_FullWave.html");
+                if (experimentTitle.equals("Diode Clipping"))
```

Comment:
Hi @akarshan96 ! Can you change the hardcoded strings with strings.xml resources? :)